app/tasks/table_source_assets.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any, Dict, List
+
+from app.crud import assets
+from app.models.pydantic.assets import AssetTaskCreate
+from app.models.pydantic.change_log import ChangeLog
+from app.models.pydantic.jobs import Job, PostgresqlClientJob
+from app.models.pydantic.metadata import DatabaseTableMetadata
+from app.tasks import get_field_metadata, writer_secrets
+from app.tasks.batch import execute
+
+
+async def table_source_asset(
+    dataset,
+    version,
+    source_uris: List[str],
+    config_options,
+    metadata: Dict[str, Any],
+    callback,
+) -> ChangeLog:
+
+    data = AssetTaskCreate(
+        asset_type="Database table",
+        dataset=dataset,
+        version=version,
+        asset_uri=f"/{dataset}/{version}/features",
+        is_managed=True,
+        creation_options=config_options,
+        metadata=DatabaseTableMetadata(**metadata),
+    )
+
+    new_asset = await assets.create_asset(**data.dict())
+
+    create_table_job = PostgresqlClientJob(
+        job_name="create table", command=["create table"], environment=writer_secrets,
+    )
+
+    load_data_jobs: List[Job] = list()
+
+    for i, uri in enumerate(source_uris):
+        load_data_jobs.append(
```

Comment:
These can all write in parallel ok? Do we need to set a limit on the number of parallel writers, or will Aurora automatically handle them properly?